app/services/SelfEmploymentService.scala

```diff
@@ -51,4 +53,11 @@ class SelfEmploymentService @Inject() (connector: SelfEmploymentConnector)(impli
     if (turnover > maxIncomeTradingAllowance) maxIncomeTradingAllowance else turnover
   }
 
+  def convertBigDecimalToMoneyString(amount: BigDecimal): String = {
```

Comment:
we can even move it to service companion object so it is a simple function as it does not have any dependencies to the service fields.